scripts/generateAssetData/ethereum/uniswapV2Pools.ts

```diff
@@ -105,7 +105,7 @@ export const getUniswapV2Pools = async () => {
         name,
         precision: Number(appTokenData.decimals),
         icons,
-        icon: 'https://assets.coincap.io/assets/icons/256/uni.png',
+        icon: 'https://coin-images.coingecko.com/coins/images/12504/large/uni.jpg?1696512319',
```

Comment:
issue: this doesn't look very asset-y 😞can we ust trust assets here? e.g this guy https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984/logo.png

Reply:
Updated in [5fa1cb5](https://github.com/shapeshift/web/pull/7298/commits/5fa1cb537de3788088524efdd98220955cb62029)